### ISSUE TITLE
brand our bots consistently

### ISSUE THREAD

**Issue by beckermr** [conda-forge/conda-forge.github.io#2384]
Our bots call themselves a few different things

- conda-forge-admin
- conda-forge-webservices[bot]
- "conda-forge-linting service"
- "conda-forge rerendering service"
- conda-forge-currator[bot]
- _others I am forgetting..._

It would be nice to decide on a consistent name and try to ensure that is used as much as possible in our tools. Some of this is unavoidable due to github account / app names, but I think we can get pretty close. I am excluding the `regro-cf-autotick-bot` from this discussion for now.

My vote for the name is `conda-forge-admin` since folks bump this account for the webservices commands.

To-do Items:

- [ ] decide on a name (likely `conda-forge-admin`)
- [ ] use the `conda-forge-admin` token as much as possible to make comments on PRs / issues / etc.
- [ ] set the git committer to `conda-forge-admin` in various spots (e.g., docker images, scripts, git CLI, github API calls, etc.)
- [ ] use the `conda-forge-admin` token as much as possible to make commits to feedstocks
- [ ] change text of comments made by bots to refer to themselves as the `conda-forge-admin`

cc @conda-forge/core

**Comment by jaimergp** [comment]
This is great and makes me happy. I think if we were to start from scratch, the obvious name would be conda-forge-bot, but that ship has sailed I guess because conda-forge-admin is way too popular now.

**Comment by beckermr** [comment]
Yeah I agree. I also think in many cases it makes sense to ask an admin to do something (eg update a team), so it's not so bad.

**Comment by jaimergp** [comment]
Works for me, then. Also, we can probably add the `[bot]` suffix to the name which gives a nice UI.

**Comment by beckermr** [comment]
Right. That bot suffix is actually added by github onto github apps, not machine accounts. Idk if we add it ourselves or if we'll need another app.

**Comment by h-vetinari** [comment]
> I think if we were to start from scratch, the obvious name would be conda-forge-bot, but that ship has sailed I guess because conda-forge-admin is way too popular now.

I would actually like to move to `conda-forge-bot`. We could even help people transition very easily: if they address `@conda-forge-admin`, we can still do the task, but tell them to please use the new™ unified™ bot-interface `@conda-forge-bot`, and by the way, here are the docs for that if you're interested.

**Comment by beckermr** [comment]
We need to figure out the status of the conda-forge-bot github account if we really think it's worth the cost of switching. It seems like a pain for very little gain IMHO.

**Comment by beckermr** [comment]
One point possibly in favor of `-admin` is that it always seems to come up first when github autocompletes users when typing `@` in an issue or PR and you get through `conda-forge-`. IDK how universal this is, but for me it is quite convenient.

**Comment by h-vetinari** [comment]
> I think if we were to start from scratch, the obvious name would be conda-forge-bot

FWIW, in the aftermath of https://github.com/conda-forge/conda-forge-bot/issues/6083, we now have https://github.com/conda-forge/conda-forge-bot 🙃 

Not sure if that makes it worth changing the bot commands, but ultimately I think that'd probably be worthwhile as a one-time price to pay for long term consistency
